file-formats/intm/type/zif_aff_intm_v1.intf.abap

```diff
@@ -0,0 +1,125 @@
+INTERFACE zif_aff_intm_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Intelligent Scenario Model</p>
+    "! Intelligent Scenario Model
+    BEGIN OF ty_islm_modelcc,
+      "! <p class="shorttext">Model Name</p>
+      "! Model Name
+      name                  TYPE c LENGTH 30,
+      "! <p class="shorttext">Parent</p>
+      "! Parent
+      parent                TYPE c LENGTH 36,
+      "! <p class="shorttext">Model Description</p>
+      "! Model Description
+      description           TYPE string,
+      "! <p class="shorttext">Model Type</p>
+      "! Model Type
+      model_type            TYPE c LENGTH 256,
+      "! <p class="shorttext">Adapter ID</p>
+      "! Adapter ID
+      adapter_id            TYPE c LENGTH 32,
+      "! <p class="shorttext">Native Model Type</p>
+      "! Native Model Type
+      native_model_type     TYPE c LENGTH 256,
+      "! <p class="shorttext">Model Content</p>
+      "! Model Content
+      model_specification   TYPE string,
```

Comment:
model_specification but title is model content. For me it is ok, but was this on purpose?

Reply:
Yes model_specification's description is model content. This is done for an internal purpose.